feature-libs/order/components/order-details/order-details-extended/consignment-tracking/consignment-tracking-link.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="cx-list-status">
+  <span>
+    <span *ngIf="consignment">
+      {{ 'orderDetails.deliveryStatus_' + consignment.status | cxTranslate }}
+    </span>
+    <span
+      *ngIf="consignmentStatus.includes(consignment.status ?? '')"
```

Comment:
[MINOR]
Instead of having logic in the HTML template, what about exposing a method for it (that is testable with unit tests), e.g. `isConsignmentStatusValid()` - it's just an example name, not sure if appropriate in this case. Maybe we could even move this method to the `OrderConsignmentService` helper service, instead of exposing the `consignmentStatus` public property

Reply:
done

Reply:
done